src/main/java/org/ohdsi/webapi/security/PermissionService.java

```diff
@@ -64,6 +64,9 @@ public class PermissionService {
     @Value("#{!'${security.provider}'.equals('DisabledSecurity')}")
     private boolean securityEnabled;
 
+    @Value("${security.ohdsi.custom.authorization.mode}")
+    private String authorizationMode;
```

Comment:
See my comment about possibly needing another config option for enabling shared artifact owner/reader permissions